Platform/Loongson/LoongArchQemuPkg/Library/MmuLib/Mmu.S

```diff
@@ -0,0 +1,157 @@
+#------------------------------------------------------------------------------
+#
+# LoongArch for LoongArch
+#
+# Copyright (c) 2021 Loongson Technology Corporation Limited. All rights reserved.<BR>
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#-----------------------------------------------------------------------------
+
+#ifndef _KERNEL
+#define _KERNEL
+#endif
+
+#include "Library/Cpu.h"
+#include "mmu.h"
+
+#  Refill the page table.
+#
+#  @param  VOID
+#
+#  @retval  VOID
+ASM_GLOBAL ASM_PFX(HandleTlbRefill)
+
+ASM_PFX(HandleTlbRefill):
+  csrwr T0, LOONGARCH_CSR_TLBRSAVE
+  csrrd T0, LOONGARCH_CSR_PGD
+  lddir T0, T0, 3   #Put pud BaseAddress into T0
+  lddir T0, T0, 2   #Put pmd BaseAddress into T0
+  lddir T0, T0, 1   #Put pte BaseAddress into T0
+  ldpte T0, 0
+  ldpte T0, 1
+  tlbfill
+  csrrd T0, LOONGARCH_CSR_TLBRSAVE
+  ertn
+ASM_GLOBAL  HandleTlbRefillEnd
+HandleTlbRefillEnd:
+
+#
+# Invalid corresponding TLB entries are based on the address given
+# @param A0 The address corresponding to the invalid page table entry
+# @retval  none
+#
+ASM_GLOBAL ASM_PFX(LoongarchInvalidTlb)

```

Comment:
It is recommended to move all ASM_GLOBAL statements to the beginning of the file, right after "#include", and check others .S file.